client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -141,7 +143,11 @@ class OrderFulfillment extends Component {
 		}
 	};
 
-	isAddressValidForLabels( address ) {
+	isAddressValidForLabels( address, type ) {
+		if ( isEnabled( 'woocommerce/extension-wcservices/international-labels' ) ) {
+			return 'destination' === type || includes( USPS_COUNTRIES, address.country );
```

Comment:
What do you think about splitting this into two functions - `isOriginAddressValidForLabels()` and `isDestinationAddressValidForLabels()`, to make it easier to understand what's being checked in each case?

Reply:
I've just tried it but I think it's more confusing. If the config flag is enabled, then the `origin` can be any of the `USPS_COUNTRIES`, and the destination can be anything. If it's not enabled, then both the origin and the destination need to be present in the `labelCountriesData` prop (which will include just US, Puerto Rico and Virgin Islands).

Would adding a few comments help?